SimCalorimetry/HcalSimProducers/src/HcalDigitizer.cc

```diff
@@ -275,6 +275,7 @@ HcalDigitizer::HcalDigitizer(const edm::ParameterSet& ps, edm::ConsumesCollector
 
   edm::ParameterSet ps0 = ps.getParameter<edm::ParameterSet>("HcalReLabel");
   relabel_ = ps0.getUntrackedParameter<bool>("RelabelHits");
+//  std::cout << "Flag to see if Hit Relabeller to be initiated " << relabel_ << std::endl;
```

Comment:
Specifically told Sunanda to wait for this PR to get merged, so shouldn't be overlapping.
This change was necessary to get the workflow going again.
Follow up should go in another PR.